### DISCUSSION TITLE
Better link name in PR body text in the footer

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#26164] [Suggest an Idea]
### Tell us more.

The Mend-hosted app PR body has this text in the footer (bottom area):

> This PR has been generated by [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/). View repository job log [here](https://developer.mend.io/github/renovatebot/renovatebot.github.io).

I'd like us to use a better link name here. Good link names are important for users who use a screen reader, as they will announce the links. But when you hear _here_, you don't know what will happen.

I don't see the code for this text in the _public_ repository, I think it's an override on the Mend hosted app side.

Here's my proposal for a better link name:

```diff
- This PR has been generated by [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/). View repository job log [here](https://developer.mend.io/github/renovatebot/renovatebot.github.io).
+ This PR has been generated by [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/). View the [repository job log](https://developer.mend.io/github/renovatebot/renovatebot.github.io).
```

After you make this change in the hosted app, also change the "Troubleshooting Renovate" docs. :wink: 

Todos for maintainer:

- [x] Update link text in Mend hosted app code/preset
- [x] Update [Troubleshooting Renovate docs](https://docs.renovatebot.com/troubleshooting/) which mention the PR body text: #30310
- [ ] Check the rest of this repo for mentions of the pr footer text, and update those

**Comment by HonkingGoose** [comment]
Can a maintainer please tell me what you think of this idea? It's been a while and I am still waiting for a reply. 🙂

**Reply by nabeelsaabna** [reply]
This is a configuration value that can be edited very quickly once we agree on a new value.
(I personally don't have any idea about such things)

**Reply by HonkingGoose** [reply]
> This is a configuration value that can be edited very quickly once we agree on a new value.

Good! So the first thing to do is to agree on a new value.

I like `repository job log`:

```markdown
View the [repository job log](https://developer.mend.io/github/renovatebot/renovatebot.github.io).
```

**Reply by nabeelsaabna** [reply]
current value:
```
"prFooter": "
 This PR has been generated by [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/). 
 View repository job log [here](https://developer.mend.io/{{platform}}/{{repository}}).
 "
```

**Reply by HonkingGoose** [reply]
```markdown
This PR was generated by [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/).
View the [repository job log](https://developer.mend.io/{{platform}}/{{repository}}).
```

I'm changing the first sentence too, by replacing `has been` with `was`. 😉

**Reply by nabeelsaabna** [reply]
done :+1:

**Reply by HonkingGoose** [reply]
Thank you!
